platform/openide.util/src/org/openide/util/ChangeSupport.java

```diff
@@ -21,7 +21,7 @@
 
 import java.util.List;
 import java.util.concurrent.CopyOnWriteArrayList;
-import java.util.logging.Level;
+import static java.util.logging.Level.*;
```

Comment:
My reasoning is:
logger.log(FINE,..... is more compact than
logger.log(Level.FILE, .......
so it reads better (phrase "Level." is a noise here).
Since log method ony accepts Level objects as first parameter there is no doubt what "FINE" is and where it comes from (well .. absolute Java beginners might have doubts the first time ... but I don't think they contribute to NetBeans).

A counter example would be turning "Paths.get("C:\.....")" into "get(C:\...")". In this case uning static import would decrease readability.